samples/grpc/restaurant-drone-deliveries-service-scala/kubernetes/service-nlb.yml

```diff
@@ -0,0 +1,17 @@
+# Network load balancer https://docs.aws.amazon.com/eks/latest/userguide/network-load-balancing.html
+apiVersion: v1
+kind: Service
+metadata:
+  name: restaurant-drone-deliveries-nlb
+  annotations:
+    service.beta.kubernetes.io/aws-load-balancer-type: external
+    service.beta.kubernetes.io/aws-load-balancer-nlb-target-type: ip
+    service.beta.kubernetes.io/aws-load-balancer-scheme: internet-facing
+spec:
+  selector:
+    app: restaurant-drone-deliveries-namespace
```

Comment:
this doesn't match with what we have in samples/grpc/shopping-cart-service-scala/kubernetes/service-nlb.yml

Reply:
Thanks, not sure how I managed to mess that up while copy-paste-replacing